Sources/SWBTaskExecution/TaskActions/CreateBuildDirectoryTaskAction.swift

```diff
@@ -28,12 +28,22 @@ public final class CreateBuildDirectoryTaskAction: TaskAction {
 
         do {
             try fs.createDirectory(directoryPath, recursive: true)
-            try fs.setCreatedByBuildSystemAttribute(directoryPath)
-            return .succeeded
         } catch {
             outputDelegate?.emitError(error.localizedDescription)
             return .failed
         }
+
+        // Warn on darwin if we are unable to set the "CreateByBuildSystem"
+        // attribute
+        do {
+            try fs.setCreatedByBuildSystemAttribute(directoryPath)
```

Comment:
The other alternative is we just use `try?` and drop the diagnostic entirely. @owenv, what do you think?

The reason I suggested the if-Darwin to @rauhul is because we'd otherwise have to change tests to ignore this message since most Linux build environments are using containers, which use tmpfs, which don't support xattrs.